src/ApiPort.VisualStudio/packages.config

```diff
@@ -0,0 +1,87 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I had tried this before and it resulted in a bunch of errors because the old MsBuild targets don't seem to know how to resolve/display it for these old csproj files.